### ISSUE TITLE
"update_only: yes" (for yum/dnf) should be enough for rule 403 to pass

### ISSUE THREAD

**Issue by egarbi** [ansible-community/ansible-lint#1277]
##### Summary
ansible-lint complain about rule 403 with the following conf:
```yaml
- name: Install patches for code-red
    yum:
      state: latest
      name: "sudo"
      update_only: yes
```

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```
ansible 2.10.4
ansible-lint 4.3.7
```

##### Desired Behaviour
ansible-lint to run smoothly and exit without errors

##### Actual Behaviour
![Screenshot from 2021-01-29 17-23-07](https://user-images.githubusercontent.com/2984201/106300573-f447fc80-6256-11eb-847a-d0c3bedcb148.png)
